Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetConnection.tsx

```diff
@@ -187,7 +187,7 @@ function AssetConnectionWindow(props: { Asset: OpenXDA.Types.Asset }): JSX.Eleme
             <div className="card-body">
                 <div style={{ width: '100%', height: '200px' }}>
                     <div style={{ height: '40px', margin: 'auto', marginTop: 'calc(50% - 20 px)' }}>
-                        <ServerErrorIcon Show={true} Size={40} Label={'A Server Error Occurred. Please Reload the Application'} />
+                        <ServerErrorIcon Show={true} Size={40} Label={'A Server Error Occurred. Please Reload the Application.'} />
```

Comment:
It may be worth moving `Label` into `gpa-gemstone` with that default, so if we do not supply a `Label` it uses this automatically

Reply:
Looks like it already has a `Label` in `gpa-gemstone`, but it's missing the final period. I'll make sure to address that when I check/update the other labels in `gpa-gemstone`.